test/suite/test_compact06.py

```diff
@@ -52,15 +52,11 @@ def test_background_compact_api(self):
                 self.session.compact(None, f'background=false,{item}'),
                 '/configuration cannot be set when disabling the background compaction server/')
 
-        #   3. We cannot disable the background server when it is already disabled.
-        self.assertRaisesWithMessage(wiredtiger.WiredTigerError, lambda:
-            self.session.compact(None, 'background=false'), '/Background compaction is already disabled/')
-
-        #   4. We cannot enable the background server with an invalid URIs to be excluded.
+        #   3. We cannot enable the background server with an invalid URIs to be excluded.
```

Comment:
Sorry, it's not you but there is a typo here URI*s*:
```suggestion
        #   3. We cannot exclue invalid URIs when enabling background compaction.
```